internal/provider/resources/block.go

```diff
@@ -27,9 +29,9 @@ type BlockResourceModel struct {
 	AccountID   customtypes.UUIDValue      `tfsdk:"account_id"`
 	WorkspaceID customtypes.UUIDValue      `tfsdk:"workspace_id"`
 
-	Name types.String         `tfsdk:"name"`
-	Type types.String         `tfsdk:"type"`
-	Data jsontypes.Normalized `tfsdk:"data"`
+	Name     types.String         `tfsdk:"name"`
+	TypeSlug types.String         `tfsdk:"type_slug"`
```

Comment:
im a little undecided on this, but i was getting confused by `type` (since that's an API object for us), so maybe we specify this as a `type_slug` (or `type-slug`) on the TF config

Reply:
oh guess they'd be underscores, based on this:

https://developer.hashicorp.com/terraform/plugin/best-practices/naming#attribute-names